src/olympia/scanners/tasks.py

```diff
@@ -478,13 +478,13 @@ def call_mad_api(all_results, upload_pk):
             score=data.get('ensemble', default_score),
```

Comment:
so, you get the `model_version` from `MAD_API_URL` results and add it the `ScannerResult(scanner=CUSTOMS)` record, but not the `ScannerResult(scanner=MAD)` record?

Reply:
Yes. This `model_version` is specific to customs results. We still have the raw response stored on the mad results, though.